lib/Scenes/Page/HomePage/Model/tutorial_model.dart

```diff
@@ -0,0 +1,15 @@
+class RecommendationCard {
+  String id;
+  String title;
+  String recomendationTittle;
+  String imageUrl;
+  String text;
+
+  RecommendationCard({
+    required this.id,
+    required this.title,
+    required this.recomendationTittle,
+    required this.imageUrl,
+    required this.text,
+  });
+}
```

Comment:
tutorial_model.dart tapi isinya recommendation card?

Reply:
Sorry my brother i will becarefull next time , i just want to make it beautifull add ',' to make it clean